package-lock.json

```diff
@@ -3729,19 +3729,22 @@
       }
     },
     "node_modules/@playwright/test": {
-      "version": "1.30.0",
-      "resolved": "https://registry.npmjs.org/@playwright/test/-/test-1.30.0.tgz",
-      "integrity": "sha512-SVxkQw1xvn/Wk/EvBnqWIq6NLo1AppwbYOjNLmyU0R1RoQ3rLEBtmjTnElcnz8VEtn11fptj1ECxK0tgURhajw==",
+      "version": "1.31.2",
+      "resolved": "https://registry.npmjs.org/@playwright/test/-/test-1.31.2.tgz",
+      "integrity": "sha512-BYVutxDI4JeZKV1+ups6dt5WiqKhjBtIYowyZIJ3kBDmJgsuPKsqqKNIMFbUePLSCmp2cZu+BDL427RcNKTRYw==",
       "dev": true,
       "dependencies": {
         "@types/node": "*",
-        "playwright-core": "1.30.0"
+        "playwright-core": "1.31.2"
       },
       "bin": {
         "playwright": "cli.js"
       },
       "engines": {
         "node": ">=14"
```

Comment:
Now Playwright 1.31 have this 'Official [docker images](https://playwright.dev/docs/docker) now include Node 18 instead of Node 16.' so bumping up playwright to 1.31 are  causing errors on CI docker images now

Reply:
Yea, it's definitely a breaking change that isn't necessary reflective in the version number for playwright/core. The only think holding this back is to adjust the peerDependencies line in the package.json to reflect the incompatibility.